src/ClashOfClans/Models/CurrentWar.cs

```diff
@@ -2,18 +2,8 @@
 
```

Comment:
This file has now unnecessary "using System;" can you please remove that also.

Reply:
using System; is required for the type DateTime

Reply:
But in CurrentWar.cs there is no DateTime properties left they are in WarBase that has the using statement.

Reply:
Yes, sorry, I thought you were talking about WarBase.cs as in your previous review but have now removed my comment and removed the unnecessary using System;